src/Tizen.Applications.Common/Tizen.Applications/AppControl.cs

```diff
@@ -45,8 +45,10 @@ public class AppControl
     {
         private const string LogTag = "Tizen.Applications";
 
-        private static Dictionary<int, Interop.AppControl.ReplyCallback> s_replyNativeCallbackMaps = new Dictionary<int, Interop.AppControl.ReplyCallback>();
-        private static int s_replyNativeCallbackId = 0;
+        private static Dictionary<int, Interop.AppControl.ReplyCallback> _replyNativeCallbackMaps = new Dictionary<int, Interop.AppControl.ReplyCallback>();
```

Comment:
Why remove `s_` prefix? it means a static variable

Reply:
I'll modify it.

Reply:
Done